pyincore/analyses/nonstructbuildingdamage/nonstructbuildingdamage.py

```diff
@@ -301,6 +300,7 @@ def building_damage_analysis_bulk_input(self, buildings):
             damage_result['hazardtype'] = hazard_type
             damage_result['hazardvals_as'] = hazard_vals_as
             damage_result['hazardvals_ds'] = hazard_vals_ds
+            damage_result['haz_expose'] = haz_expose
```

Comment:
I can't test this since NFS is down, but I think haz_expose should be part of the building_result. That's where we put it for building damage.

Reply:
ohh good catch! Will update that